- playlist editing and viewing has been created.

dencharski opened the pull request.

dencharski committed:
- playlist editing and viewing has been created.

kosstyango commented:
⚠️ Критические замечания:
1. Поиск начинается не через 2 секунды после ввода последнего знака, а сразу после ввода первого - это неправильно.
2. Всплывающее меню "Добавить в плейлист" на экране "Аудиоплеер" всплывает низко: видно только один плейлист.
3. При входе в аудиоплеер из экрана "Медиатека" экран аудиоплеера сохраняется после фонового режима. А если заходить в аудиоплеер из результатов поиска - то при восстановлении из фона мы окажемся в "Избранных треках" Медиатеки :(

🍏 Рекомендации, не влияющие на результат проверки работы:
Дефекты прошлых спринтов:
    Курсор в поле поиска должен быть синий, а не серый (Спринт 8).
    Боттом-меню на экране поиска показывается над клавиатурой - это не соответствует макету в Фигме (Спринт 19).
    У всех треков в строке "Страна" указана USA - даже у русских народных

https://github.com/dencharski/Playlist-Maker/assets/73995013/0f223ea4-bbfa-4dde-9cfb-48195f0c884a

dencharski committed:
- playlist editing and viewing has been created.
-fix search track,
-resize bottomSheet(add in playlist),
-fixed screen collapsing and restoring
